src/RingGrids/interpolation.jl

```diff
@@ -213,9 +213,9 @@ function interpolate(   A::AbstractGrid{NF},        # field to interpolate
                         I::AbstractInterpolator     # indices in I are assumed to be calculated already!
                         ) where NF                  # use number format from input data also for output
 
-    (; npoints ) = I.locator             # number of points to interpolate onto
-    Aout = Vector{NF}(undef, npoints)        # preallocate: onto θs, λs interpolated values of A
-    interpolate!(Aout, A, I)                  # perform interpolation, store in As
+    (; npoints ) = I.locator                # number of points to interpolate onto
+    Aout = Vector{NF}(undef, npoints)       # preallocate: onto θs, λs interpolated values of A
```

Comment:
The interpolation is still WIP, right? 

Because here and in the following it would need a more flexible constructor, a `similar` or a `ArrayType{...}(undef, ...)` or something similar

Reply:
Yes, interpolation is currently only used for output to get from reduced grids to full grids or for the interactive visualisation with `plot`. The only place where interpolation takes place in the dynamical core is the particle advection and that's passive and switched off by default. So I'm tending towards postponing a GPU-ready version of the interpolation for when we actually need it.

Reply:
Ok, agreed!